src/SFA.DAS.EmployerApprenticeshipsService.Infrastructure/Data/DasLevyRepository.cs

```diff
@@ -251,7 +252,7 @@ public DasLevyRepository(LevyDeclarationProviderConfiguration configuration, ILo
             return result;
         }
 
-        public async Task CreatePaymentData(IEnumerable<PaymentDetails> payments)
+        public async Task CreatePayments(IEnumerable<PaymentDetails> payments)
```

Comment:
This method could do with a little bit of love.  Maybe extract out a few private methods?

Reply:
Done